Tests/fixtures/tablet.yml

```diff
@@ -9595,10 +9675,250 @@
   device:
     type: tablet
     brand: LE
-    model: IdeaTab B8000-F
+    model: Yoga Tablet 10
   os_family: Android
   browser_family: Android Browser
-- 
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.2.2; pl-pl; Lenovo B8000-H/JDQ39) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.2.2 Mobile Safari/534.30
+  os:
+    name: Android
+    short_name: AND
+    version: "4.2.2"
+    platform:
+  client:
+    type: browser
+    name: Android Browser
+    short_name: AN
+    version:
+    engine: WebKit
+    engine_version: "534.30"
+  device:
+    type: tablet
+    brand: LE
+    model: Yoga Tablet 10
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.2.2;pl-pl; Lenovo B8000-F/JDQ39) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.2.2 Mobile Safari/534.30
```

Comment:
They are almost the same but still some differences (some rules are expecting "Build" sometimes so thought it's worth leaving those)
Lenovo B8000-F Build/JDQ39
Lenovo B8000-H/JDQ39
Lenovo B8000-F/JDQ39